guides/common/modules/ref_resource-quota-settings.adoc

```diff
@@ -0,0 +1,8 @@
+[id="resource-quota-settings"]
+= Resource Quota settings
+
+The Resource Quota plugin contains the following global {Project} settings:
```

Comment:
```suggestion
You can configure the following global {Project} settings for the Resource Quota plug-in:
```

Reply:
Can you please add a sentence about navigation?
```
You can find the Resource Quota settings by navigating to *Administer* > *Settings* > *<tab name>*.
```

It can go at the bottom.